Weather/Weather/Scenes/Base/BaseTableViewController.swift

```diff
@@ -32,6 +33,8 @@ class BaseTableViewController: UIViewController {
             $0.delegate = self
         }
     }
+    
+    func configView() { }
```

Comment:
```suggestion
    private func configView() { }
```
với những func chỉ sử dụng trong nội bộ của scope thì thêm từ khoá `private`